### ISSUE TITLE
Add SQL binding command should be available on file explorer context menu

### ISSUE THREAD

**Issue by Charles-Gagnon** [microsoft/azuredatastudio#17731]
It would be handy to have it show up on right click for cs files instead of just through command palette

**Comment by lucyzhang929** [comment]
@Charles-Gagnon @chlafreniere @dzsquared Thoughts on the ordering for "Add SQL Binding" in the context menu?
<img width="466" alt="Screen Shot 2021-11-30 at 7 45 57 PM" src="https://user-images.githubusercontent.com/23462877/144168989-d7f2d53f-7f26-4227-8e51-891a1ac7008b.png">

**Comment by Charles-Gagnon** [comment]
Something like that looks great @lucyzhang929 

What @dzsquared said would be good though, if there's a way to make this only show up when we detect that it's a functions project that would be great. I think you might be able to achieve something like this using a custom context and the 'in' clause : https://code.visualstudio.com/api/references/when-clause-contexts#in-conditional-operator

(there may be a better way of doing it though, feel free to look through the options there and see if you can think of any other ways to get what we want)

This might be something we punt though - that's probably going to be a bit more work to hook it all up and while useful it isn't something we need right away.

**Comment by lucyzhang929** [comment]
Yep right now the only condition we check is that the file is csharp so it can be annoying/confusing for users not in a functions project. I'll check if theres an easy way to hook up the custom context, if not we can punt this.
